python/ray/worker.py

```diff
@@ -1790,6 +1795,9 @@ def init(redis_address=None,
         include_webui: Boolean flag indicating whether to start the web
             UI, which is a Jupyter notebook.
         use_raylet: True if the new raylet code path should be used.
+        logging_level: Logging level, default will be loging.WARN.
+        logging_format: Logging format, default will be "%(message)s"
+            which means only contains the message.
```

Comment:
I think it would be useful also to add `configure_logging` argument, which would be by default `True`, but that other programs which call `ray.init` can disable it because they configure logging on their own.

Reply:
This is a good feature, especially when the user wants to modify the format.